services/user.js

```diff
@@ -326,6 +328,17 @@ class User {
                 newProfile[key] = updatedUser[key];
             });
 
+            const aUser = user[0];
+            const isUserActivated = (aUser.userStatus) && (aUser.userStatus != USER.STATUSES.INACTIVE);
```

Comment:
!== should be used here

The ?. operator can be used for this to remove the separated null check:
`const isUserActivated = aUser?.userStatus !== USER.STATUSES.INACTIVE);`

Reply:
updated!